plugins/f1-scoreboard/f1_renderer.py

```
@@ -77,6 +78,16 @@ def _load_fonts(self) -> Dict[str, Any]:
 
         return fonts
 
+    def _to_local_dt(self, utc_iso_str: str) -> datetime:
+        """Parse a UTC ISO datetime string and convert to configured local timezone."""
+        dt = datetime.fromisoformat(utc_iso_str.replace("Z", "+00:00"))
+        tz_str = self.config.get("timezone", "UTC")
+        try:
+            local_tz = pytz.timezone(tz_str)
+        except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)